features/features.go

```diff
@@ -53,6 +53,8 @@ const (
 	DbusActivation
 	// HiddenSnapDataHomeDir controls if the snaps' data dir is ~/.snap/data instead of ~/snap
 	HiddenSnapDataHomeDir
+	// MoveSnapHomeDir controls whether snap user data under ~/snap (or ~/.snap/data) are moved to ~/Snap.
```

Comment:
Good point, I changed it to "can be moved". Thanks

Reply:
Mmm... then I probably still don't understand what this flag is for :-) If it's set to true, does it mean that a migration will happen for sure, or will it only happen if some other configuration option is set?

Reply:
Oh sorry, I forgot to mention that in this PR's description. The idea is that there will be this feature flag and a separate debug command to trigger the change. The flag is there so that the user knows this is experimental. Then the change will be triggered by `snap debug migrate-home <snaps>`.